src/test/java/se/michaelthelin/spotify/MiscTest.java

```diff
@@ -0,0 +1,16 @@
+package se.michaelthelin.spotify;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class MiscTest {
```

Comment:
```suggestion
public class Base64Test {
```

Reply:
I called it MiscTest because it seems like there wasn't any "everything else" test class yet. But sure, will change

Reply:
Ok! In the best case push the changes and I'll check if my threads can be resolved. I mean in this case I totally trust that you understand the suggestion the same way as I do, but you know, in general it's good to have the thread issuer decide if the changes are satisfying / understood in the same way :v: